### PR TITLE
Add customerDocumentCustomField setting to dLocal

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by dara123 — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/dLocal.yaml [hunk 1]
```diff
@@ -42,3 +42,8 @@ allOf:
             type: boolean
             default: false
             description: Specifies whether to create an installment plan and use it for payment.
+          customerDocumentCustomField:
+            type: string
+            description: |-
+              Name of the transaction custom field to use for sending customer document.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccountConfig/dLocal.yaml:48]: suggestion: I think this reads better if we place the adjective before the object 
```suggestion
              Name of the custom transaction field to use to send the customer document.
```

File: openapi/components/schemas/GatewayAccountConfig/dLocal.yaml [hunk 2]
```diff
@@ -42,3 +42,8 @@ allOf:
...
+              Use this setting to skip intermediate step of collecting customer document for payment card transactions.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccountConfig/dLocal.yaml:49]: nitpick:
```suggestion
              Use this setting to skip the intermediate step of collecting customer the document for payment card transactions.
```

Comment (ashkarpetin) [reply openapi/components/schemas/GatewayAccountConfig/dLocal.yaml:49]: > of collecting customer the document 

That doesn't seem correct

File: openapi/components/schemas/GatewayAccountConfig/dLocal.yaml (hunk 1 above)
Comment (ashkarpetin) [reply openapi/components/schemas/GatewayAccountConfig/dLocal.yaml:48]: That seems incorrect that's not a custom transaction field, that's a field which is taken from transaction.customFields

Comment (ashkarpetin) [reply openapi/components/schemas/GatewayAccountConfig/dLocal.yaml:48]: I think the semantics changes.

`Custom field of the transaction` (customFields is a property on transaction resource) vs `Custom transaction field` (which looks like any propery on transaction)

**Review by 2e3s — APPROVED** [review]

**Review by colinbird — APPROVED** [review]

File: openapi/components/schemas/GatewayAccountConfig/dLocal.yaml (hunk 1 above)
Comment (noon-dawg) [reply openapi/components/schemas/GatewayAccountConfig/dLocal.yaml]: I think your update on this description was perfect! Thanks for taking my input onboard 🙏

**Pull request merged**